src/main/java/com/aws/greengrass/easysetup/GreengrassSetup.java

```diff
@@ -519,7 +519,7 @@ void provision(Kernel kernel) throws IOException, DeviceConfigurationException {
 
     @SuppressWarnings("PMD.PreserveStackTrace")
     private void setComponentDefaultUserAndGroup(DeviceConfiguration deviceConfiguration) {
-        if (Exec.isWindows) {
+        if (PlatformResolver.isWindows) {
```

Comment:
`Exec.isWindows` was added into the CLI by Francis, let's replace that with a Platform lookup since you're moving exec.

Reply:
https://github.com/aws-greengrass/aws-greengrass-cli/pull/142